Added header to manage user page

wanawsha opened the pull request:
### What does this PR do?
- Added header to manage user page

### Any background context you want to provide?
- Had some issues with adding a green underline on hover. Tried to copy from another html page, but it didn't want to work.

Hacer2780 commented:
Hi @wanawsha 
![image](https://github.com/user-attachments/assets/3eab5b85-d11c-4d7c-9ea0-2e97e394909b)
You added the header to the Manage User page, but it doesn't fully match the Figma design.
Could you please review the spacing and font weights?
Also, the links in the header are not functional
https://www.figma.com/design/IdBpLb1ou1CH8TY716G49E/Edugate?node-id=0-1&p=f&t=Yy0MUiM0QiQGtfQG-0
Thank you

MarcoSaba71ni committed:
draft-admin-profile page

In order not to create a disorder inside the folders, the file that would better match the Figma design would be draft-admin-profile-page.

MarcoSaba71ni committed:
new admin-profile page

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-2-basic-layout-manage-user-student-AnnaK

inastefansdottir committed:
Merge pull request #598 from NoroffFEU/epic-5-issue-558-fredrikborgstein

Epic 5: Add accessability to teacher edit profile form

JKollerud committed:
Merge branch 'develop' into 425-jk-fix

ReZvanYY committed:
Update modals.html

Update commit

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-2-basic-layout-manage-user-student-AnnaK

saimoh03 committed:
meging to develop

saimoh03 committed:
resolve conflict of add-teacher.html

saimoh03 committed:
hashtag added in add-teacher-page

saimoh03 committed:
recover edit-profile-teacher temeplete

saimoh03 committed:
recovered edit-profile-teacher temeplete

saimoh03 committed:
conflict resolved

saimoh03 committed:
conflict resolved & remove issues

YeabseraG committed:
Update edit-profile-student.html

saimoh03 committed:
dialog box added on delete

saimoh03 committed:
user-student updated

saimoh03 committed:
real-time search work & delete works as per instructions

Amf1b1an committed:
Changes to main, section, divs and button

Changes to main, section, divs and button in order to achieve pixelperfect button

saimoh03 committed:
sidebar hover added

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-1-basic-layout-w.a

rscode67 committed:
correct link to 'back-to-dashboard'

neginfarzib committed:
Update select-user.html

sidebar-navbar

JKollerud committed:
stylesheet and script

JKollerud committed:
Merge branch '425-jk-fix' of https://github.com/NoroffFEU/edu-gate into 425-jk-fix

Malin-Malin committed:
implemented accessibility features

ReZvanYY committed:
Update modals.html

Updated commit

JKollerud committed:
logo md size

JKollerud committed:
breadcrumb margin

JKollerud committed:
hamburger, margin form

JKollerud committed:
footer img size

JKollerud committed:
breadcrumb margin md

JKollerud committed:
margin footer

Martine04384 committed:
Updated footer img sm screens and filepath logo.

Martine04384 committed:
Updated hamburger icon.

TonjeSchjefstad committed:
Profile Image Update

TonjeSchjefstad committed:
Updated header for admin-profile.html

Malin-Malin committed:
Making the page responsive and adjusting the padding to match the Figma design.

Malin-Malin committed:
Replacing the current header with the 'Top-Performing Student' header due to responsiveness issues.

Malin-Malin committed:
Adjusting margins and padding for the header, logo, and top areas of the page.

Malin-Malin committed:
Made the header border visible, as it appears in both the mobile and desktop designs in Figma.

Martine04384 committed:
Added responsiveness

The page is now responsive but breaks on md screens because of the header. Did some adjustments on desktop to make it look even more like the figma design.

Malin-Malin committed:
corners from rounded to rounded-xl

mamf92 committed:
Fixed image at bottom of page, added link to logo, corrected padding in input fields, checked functionality of page.

GustavBoe committed:
Merge pull request #627 from NoroffFEU/522-ticket-17-implement-accessibility-features

Implemented accessibility features. Authored by Malin-Malin.

GustavBoe committed:
Merge branch 'develop' into 534-profile-image

GustavBoe committed:
Merge pull request #628 from NoroffFEU/534-profile-image

Profile Image Update. Authored by TonjeSchjefstad.

GustavBoe committed:
Merge branch 'develop' into 533-header

GustavBoe committed:
Merge pull request #629 from NoroffFEU/533-header

Updated header for admin-profile.html. Authored by TonjeSchjefstad.

ReZvanYY committed:
Update modals.html

cleared temporal code.

ReZvanYY committed:
Merge branch 'develop' into implement-the-student-result-edit-modal-546-MR

GustavBoe committed:
Merge branch 'develop' into 521-ticket-16-implement-form-layout-and-responsiveness

GustavBoe committed:
Merge pull request #631 from NoroffFEU/521-ticket-16-implement-form-layout-and-responsiveness

521 ticket 16 implement form layout and responsiveness. Authored by Malin-Malin.

ReZvanYY committed:
Merge branch 'develop' into implement-the-student-result-edit-modal-546-MR

ReZvanYY committed:
Hot fixes

Noticed minor issues that needed to be fixed for the code to be pixel perfect.

ReZvanYY committed:
Update modals.html

Minor updates needed.

ReZvanYY committed:
Merge pull request #633 from NoroffFEU/implement-the-student-result-edit-modal-546-MR

Implement the student result edit modal 546 mr

This merges the modal for the student result edit template into the develop branch for the Edu-Gate project at Noroff.

MikkelAugust committed:
Merge branch 'develop' into 617-fix-success-alert

mamf92 committed:
Started work on a merged version of the index page currently called new-index.html. This will be removed later an current. Also added logo which will not be included in the merge because it is already part of an earlier pull request.

mamf92 committed:
Minor changes to current index page. Will change further later.

mamf92 committed:
Merge branch 'develop' into 392-user-story-3-implement-a-pixel-perfect-signup-page-for-mobile-and-tablets

rscode67 committed:
Merge branch 'develop' into epic-6-user-story-3-add-upload-school-page-annak

Malin-Malin committed:
rounded-lg

JKollerud committed:
Merge branch 'develop' into 425-jk-fix

Malin-Malin committed:
Hidden field on mobile devices.

mamf92 committed:
Finall check of functionality and responsivness done. Mobile, tablet, and desktop page now matches Figma design file.

ReZvanYY committed:
Update modals.html

The X was stretched out, fixed the X

ReZvanYY committed:
Update modals.html

changed the height and width of the X

ReZvanYY committed:
Update sign-up.html

Hid the alert pop up.
Made the logo be clickable and linked to index.html

mamf92 committed:
Merge branch 'develop' into 573-epic-1-fixing-landing-page

Martine04384 committed:
Updated logo header, small adjustments hamburger menue, updated margins around breadcrumbs according to figma.

Martine04384 committed:
Updated space around breadcrumbs

Martine04384 committed:
Final update breadcrumbs.

Martine04384 committed:
Final update breadcrumbs.

Martine04384 committed:
Breadcrumbs updated on sm and lg screens!

inastefansdottir committed:
Merge pull request #630 from NoroffFEU/419-new-responsive-teacher-profile-page

Issue 419 - Responsive teacher-profile.html

Martine04384 committed:
Updated logo size desktop, hamburger icon, padding breadcrumbs, size main "box" and padding inside main, padding top h1, padding bottom button.

inastefansdottir committed:
Merge branch 'develop' into 416-Fix-margins-and-logo-sm-screens

Martine04384 committed:
Updated nav with same nav as edit-admin-profile, moved bredcrumbs into main.

inastefansdottir committed:
Merge pull request #577 from NoroffFEU/416-Fix-margins-and-logo-sm-screens

Updated logo, hamburger icon, margins header and main.

rscode67 committed:
Merge branch 'develop' into epic-6-user-story-3-add-upload-school-page-annak

MarcoSaba71ni committed:
Final Changes

mamf92 committed:
Merge branch 'develop' into 573-epic-1-fixing-landing-page

mamf92 committed:
Merge pull request #644 from NoroffFEU/573-epic-1-fixing-landing-page

573 epic 1 fixing landing page
Redid landing page to ensure it matches design file on both mobile and desktop, and that the pages are responsive and all functionality works.

mamf92 committed:
Merge branch 'develop' into 392-user-story-3-implement-a-pixel-perfect-signup-page-for-mobile-and-tablets

mamf92 committed:
Merge pull request #632 from NoroffFEU/392-user-story-3-implement-a-pixel-perfect-signup-page-for-mobile-and-tablets

Completed task 392 user story 3 implement a pixel perfect signup page for mobile and tablets. Page should now be pixel perfect for both mobile, and desktop, with a responsiveness from 393px to 1440px with all functionality working.

JKollerud committed:
removed script

JKollerud committed:
Merge branch '425-jk-fix' of https://github.com/NoroffFEU/edu-gate into 425-jk-fix

JKollerud committed:
Footer img

JKollerud committed:
breadcrumb and logo alignment

JKollerud committed:
Merge branch 'develop' into 425-jk-fix

JKollerud committed:
Border mobile closer

JKollerud committed:
Merge branch '425-jk-fix' of https://github.com/NoroffFEU/edu-gate into 425-jk-fix

Martine04384 committed:
Merge branch 'develop' into 613-fix-nav-teacher-profile

ReZvanYY committed:
Merge branch 'develop' into 637-epic-1-fix-signup-page-desktop-MR

Martine04384 committed:
Updated the branch and made some small adjustments in the header.

JKollerud committed:
nav md smaller

JKollerud committed:
padding main left right

JKollerud committed:
footer responsive

JKollerud committed:
margin footer desktop

JKollerud committed:
input fields pl

saimoh03 committed:
Merge pull request #450 from NoroffFEU/user-story-2-add-side-bar-add-teachers-page-AnnaK

made sidebar responseve one hover, & added sidebar to 4 templetes

ReZvanYY committed:
Delete This Student's Results

Delete This Student's Results for mobile & tablet view has been completed.

ReZvanYY committed:
Merge branch 'develop' into 637-epic-1-fix-signup-page-desktop-MR

ReZvanYY committed:
Minor fixes

Found minor issues. Fixed the issues

ReZvanYY committed:
Merge branch '637-epic-1-fix-signup-page-desktop-MR' of https://github.com/NoroffFEU/edu-gate into 637-epic-1-fix-signup-page-desktop-MR

ReZvanYY committed:
Merge pull request #643 from NoroffFEU/637-epic-1-fix-signup-page-desktop-MR

Hot fixes have been approved merging.

AndersenIA committed:
Merge branch 'develop' into 640-epic-1-fix-success-modal-MR

mamf92 committed:
Looked at font-size issues. No issues found. Fixed use of px in sizing of buttons.

mamf92 committed:
Merge branch 'develop' into 639-epic-1-fix-landing-page-mobile

ReZvanYY committed:
fix-success-modal

The X was unknown for desktop view. Equation fixed.

ReZvanYY committed:
Merge branch '640-epic-1-fix-success-modal-MR' of https://github.com/NoroffFEU/edu-gate into 640-epic-1-fix-success-modal-MR

ReZvanYY committed:
Files were successfully added!

Updated the X.

ReZvanYY committed:
Merge pull request #642 from NoroffFEU/640-epic-1-fix-success-modal-MR

The case of the X was solved.

martir2109 committed:
fixed the the navigation bar to look like the other pages such as About, contact, etc. And fixed som of the links

mamf92 committed:
Fixed font size on heading and main text. Fixed about and contact button to be pixel perfect.

inastefansdottir committed:
Merge branch 'develop' into 613-fix-nav-teacher-profile

JKollerud committed:
Merge branch 'develop' into 425-jk-fix

JKollerud committed:
removed error

inastefansdottir committed:
Merge pull request #647 from NoroffFEU/613-fix-nav-teacher-profile

Updated nav and breadcrumbs

ReZvanYY committed:
Issue: #547 Completed

The implementation of "Delete This Student's Results?" has now been completed.

ReZvanYY committed:
Merge branch 'develop' into 547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH

mamf92 committed:
Merge branch 'develop' into 639-epic-1-fix-landing-page-mobile

martir2109 committed:
Merge remote-tracking branch 'origin/develop' into epic-6-user-story-1-add-upload-student-annak

GustavBoe committed:
Merge branch 'develop' into 526-ticket-25-create-non-editable-input-fields

GustavBoe committed:
Merge branch 'develop' into 526-ticket-25-create-non-editable-input-fields

GustavBoe committed:
Merge pull request #576 from NoroffFEU/526-ticket-25-create-non-editable-input-fields

526 ticket 2.5 create non editable input fields. Authored by Malin-Malin.

ReZvanYY committed:
Minor fixes.

Discovered minor issues in the code. Fixed the issues.

ReZvanYY committed:
Merge branch '547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH' of https://github.com/NoroffFEU/edu-gate into 547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH

ReZvanYY committed:
Merge branch 'develop' into 547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH

ReZvanYY committed:
minor updates

Minor updates for the code to be pixel perfect.

ReZvanYY committed:
Merge branch '547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH' of https://github.com/NoroffFEU/edu-gate into 547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH

JKollerud committed:
removed margin breadcrumb

JKollerud committed:
hamburger size

JKollerud committed:
logo breadcrumb alignment

JKollerud committed:
Merge branch 'develop' into 425-jk-fix

YeabseraG committed:
update ticket 2.2, 2.3

px changed to rem
lg-px removed, input fields no longer change with size.
2nd border deleted

MarcoSaba71ni committed:
Unfinished changes

Please, read the last comments

AndersenIA committed:
Fixing the modal

Fixed the modal and made it pixel perfect.

AndersenIA committed:
Merge branch '547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH' of https://github.com/NoroffFEU/edu-gate into 547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH

ReZvanYY committed:
Merge branch 'develop' into 617-fix-success-alert

jhanArvie committed:
Validation

Small changes for contact-error.html and contact.html
Made a test page for error validation of contact page named "error-test.html"
if approve this will be move to contact.html

ReZvanYY committed:
Merge pull request #650 from NoroffFEU/547-epic-9-user-story-4-implement-the-student-result-delete-modal-MRH

Delete This Student's Results has now been implemented.

ReZvanYY committed:
Minor fixes

Minor fixes.

ReZvanYY committed:
minor fixes

Minor fixes to the hamburger menu

ReZvanYY committed:
Merge pull request #653 from NoroffFEU/638-epic-1-fix-registration-page-MRH

Minor fixes needed to make the code pixel perfect.

ReZvanYY committed:
Merge branch 'develop' into 617-fix-success-alert

GustavBoe committed:
Merge branch 'develop' into admin-profile-page

GustavBoe committed:
Merge branch 'develop' into admin-profile-page

GustavBoe committed:
Merge pull request #620 from NoroffFEU/admin-profile-page

Admin profile page. Authored by MarcoSaba71ni

jhanArvie committed:
validation added

moved the test (finished) validation page to original contact.html and delete the test page

kevindjenhansen committed:
Merge pull request #652 from NoroffFEU/validation/contacts-error-input

Validation

Amf1b1an committed:
Changes to Edit Profile Button

JKollerud committed:
Merge branch 'develop' into 425-jk-fix

saimoh03 committed:
Merge branch 'develop' into epic-6-user-story-2-creating-upload-in-code

saimoh03 committed:
add teacher border fix

mamf92 committed:
Merge branch 'develop' into 639-epic-1-fix-landing-page-mobile

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-2-basic-layout-manage-user-student-AnnaK

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-1-basic-layout-w.a

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story1-manage-user-search-page-AnnaK

GustavBoe committed:
Merge branch 'develop' into T3.6-Update-Button-E4

GustavBoe committed:
Merge pull request #654 from NoroffFEU/T3.6-Update-Button-E4

T3.6 update button e4. Authored by Amf1b1an.

saimoh03 committed:
Merge pull request #582 from NoroffFEU/epic-7-user-story-2-basic-layout-manage-user-student-AnnaK

Epic 7 user story 2 basic layout manage user student annaK

martir2109 committed:
added alt tag to some of the images/ icons and a title tag to a button

martir2109 committed:
removed font-semibold from nav ul class and added font-roboto

martir2109 committed:
added the correct navigation bar that is the same as dashboard, about and contact with the hovering green line

martir2109 committed:
added div flex flex-col gap-12 mt-10 in the sidebar to push the icons futher down like in the figma file, but making the grey right border stay fixed

martir2109 committed:
added font-roboto in the body tag

Hacer2780 commented:
Hi 
@wanawsha @kaijanna 
Many people outside the group have commented on the work @wanawsha  is working on. @wanawsha thinks about this situation as follows: «I tried to create a branch but I assume I did it incorrectly. So everyone’s committing to that branch. Don’t know what happened. And I didn’t want to remove it in case everything got deleted.» What can we do about this @sulenchy? 
Thank you for the help😊

JKollerud committed:
revert changes

malintheier committed:
Added changes

JKollerud committed:
Merge branch 'develop' into 425-jk-fix

JKollerud committed:
Indentation css 2

YeabseraG committed:
Final Update (hopefully)

style changes for input fields

JKollerud committed:
Merge branch '425-jk-fix' of https://github.com/NoroffFEU/edu-gate into 425-jk-fix

MikkelAugust committed:
fixed success alert

made it more pixle perfect and responsive.

MikkelAugust committed:
Merge branch 'develop' into 617-fix-success-alert

mamf92 committed:
Merge branch 'develop' into 639-epic-1-fix-landing-page-mobile

fredrikborgstein committed:
Fix: Added accessability to the teacher profile page, with proper aria labels

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-1-basic-layout-w.a

martir2109 committed:
Added font-roboto to the body tag. Added the correct navigation bar/ sidebar so that the borders/ lines in the left top corner are correct.

Shamia702 committed:
i run the code on validator and fixed all the errors and warnings.

MikkelAugust committed:
Merge pull request #634 from NoroffFEU/617-fix-success-alert

Fixed Success alert responsiveness

moni999fd committed:
Fixed accessibility issues in edit-profile-admin.html

martir2109 committed:
Fixed the two buttons, update and delete user to be more similar to the figma file. Changed space-x-4 to space-x-12, spacing between the buttons. Added 1.5rem text to ensure the text is the correct sizing. Whitespace-nowrap to ensure the text stays on one line and not wrapping to the next line. Also added font-semibold and font-poppins.

martir2109 committed:
Added md to rounded in the input fields to ensure it is the same as the figma file, rounded-md applies 6px border radius

inastefansdottir committed:
Merge branch 'develop' into 425-jk-fix

inastefansdottir committed:
Merge pull request #571 from NoroffFEU/425-jk-fix

425 jk fix

GustavBoe committed:
Merge branch 'develop' into epic-4--ticket-2.2,-2.3,-2.5

GustavBoe committed:
Merge pull request #615 from NoroffFEU/epic-4--ticket-2.2,-2.3,-2.5

ticket 2.2, 2.3 for review. Authored by YeabseraG.

inastefansdottir committed:
Merge branch 'develop' into task-3-shamia-admin-upload

inastefansdottir committed:
Fixed some conflict errors that affected the styling of the page

inastefansdottir committed:
Merge pull request #658 from NoroffFEU/task-3-shamia-admin-upload

updated edit-profile-admin.html

inastefansdottir committed:
Merge branch 'develop' into task-557-monica-accessibility-fix

inastefansdottir committed:
Fixed minor errors in the code

inastefansdottir committed:
Merge pull request #659 from NoroffFEU/task-557-monica-accessibility-fix

Fixed accessibility issues in edit-profile-admin.html

inastefansdottir committed:
Merge branch 'develop' into Epic5-teacherprofile-accessability-fredrikborgstein

sanderedvardsen committed:
Started styling sidebar and updated dependencies

sanderedvardsen committed:
Merge branch 'epic-7-user-story-2-edit-student-page-annak' of https://github.com/NoroffFEU/edu-gate into epic-7-user-story-2-edit-student-page-annak

sanderedvardsen committed:
changed background color on main

martir2109 committed:
removed font-semibold from dashboard > select user > manage user as this is more similar to the figma file

martir2109 committed:
I undid the font-semibold removal — it was actually supposed to be there, I just saw it wrong.

fredrikborgstein committed:
Fix: Wrapped the div that errored out on the validator with a li, to get proper structure.

inastefansdottir committed:
Merge pull request #657 from NoroffFEU/Epic5-teacherprofile-accessability-fredrikborgstein

Fix: Added accessibility to the teacher profile page, with proper ari…

saimoh03 committed:
css updated

saimoh03 committed:
css updated v1.0

saimoh03 committed:
Merge branch 'develop' into epic-6-user-story-2-creating-upload-in-code

rscode67 committed:
Merge pull request #626 from NoroffFEU/epic-6-user-story-3-add-upload-school-page-annak

epic-6-user-story-3-add-upload-school-page-annak

saimoh03 committed:
Merge branch 'develop' into epic-6-user-story-2-creating-upload-in-code

sanderedvardsen committed:
added update successful modal and fixed forms with forms validation

Shamia702 committed:
Update edit-profile-teacher.html

just did my part nothng else i changes, hopefully it should work now

saimoh03 committed:
Merge pull request #540 from NoroffFEU/epic-6-user-story-2-creating-upload-in-code

Epic 6 user story 2 creating upload in code

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story1-manage-user-search-page-AnnaK

saimoh03 committed:
css updated

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-1-basic-layout-w.a

martir2109 committed:
Merge remote-tracking branch 'origin/develop' into epic-6-user-story-1-add-upload-student-annak

martir2109 committed:
added the upload error and verified alert box from the files add-teacher-fail.html and add-teacher-verifired.html, but moved it to be on the right side of the page on the add-student-new.html file

martir2109 committed:
added the correct upload from the file add-teacher.html into add-students-new.html

inastefansdottir committed:
Merge branch 'develop' into task-12-shamia-edit-teacher-profile-new

inastefansdottir committed:
Merge pull request #661 from NoroffFEU/task-12-shamia-edit-teacher-profile-new

Update edit-profile-teacher-new-branch.html

martir2109 committed:
Merge branch 'develop' into epic-6-user-story-1-add-upload-student-annak

rscode67 committed:
add upload to school page

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-1-basic-layout-w.a

sanderedvardsen committed:
Merge branch 'epic-7-user-story-2-edit-student-page-annak' of https://github.com/NoroffFEU/edu-gate into epic-7-user-story-2-edit-student-page-annak

inastefansdottir committed:
Fixed minor styling errors to make the pages pixel perfect according to the figma file, and bug fixed the code

martir2109 committed:
added the sidebar div again, seems it had disappeared

martir2109 committed:
Changed the success message to User was successfully updated

martir2109 committed:
Merge branch 'develop' into epic-7-user-story-2-edit-student-page-annak

inastefansdottir committed:
Merge pull request #667 from NoroffFEU/641-bug-fix-ina

Fixed minor styling errors to make the pages pixel perfect according to the figma file

martir2109 committed:
Merge pull request #668 from NoroffFEU/epic-7-user-story-2-edit-student-page-annak

Epic 7 user story 2 edit student page annak

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story-1-basic-layout-w.a

saimoh03 committed:
Merge branch 'develop' into epic-7-user-story1-manage-user-search-page-AnnaK

martir2109 committed:
Merge pull request #662 from NoroffFEU/epic-6-user-story-1-add-upload-student-annak

Epic 6 user story 1 add upload student annak

saimoh03 committed:
Merge pull request #625 from NoroffFEU/epic-7-user-story-1-basic-layout-w.a

sidebar hover added

saimoh03 committed:
Merge pull request #624 from NoroffFEU/epic-7-user-story1-manage-user-search-page-AnnaK

Epic-7-user-story1-manage-user-search-page-AnnaK

rscode67 committed:
Merge pull request #666 from NoroffFEU/Epic-6-user-story-3-add-upload-school

Epic 6 add upload to school page annak

mamf92 committed:
Merge branch 'develop' into 639-epic-1-fix-landing-page-mobile

mamf92 committed:
Merge pull request #649 from NoroffFEU/639-epic-1-fix-landing-page-mobile

malintheier committed:
Merge branch 'develop' into 605-set-up-a-start-scriptserver-to-run-template-html-files-with-live-server

malintheier committed:
Update package.json

Co-authored-by: Abudu Abidun Sulaiman <sulaimanabiodun172@gmail.com>

malintheier committed:
Update package.json

Co-authored-by: Abudu Abidun Sulaiman <sulaimanabiodun172@gmail.com>

sulenchy committed:
Merge pull request #656 from NoroffFEU/605-set-up-a-start-scriptserver-to-run-template-html-files-with-live-server

Set up live-server to serve HTML files with auto-reloading on changes

malintheier committed:
redirecting to index

malintheier committed:
Update package.json

malintheier committed:
Update package.json

malintheier committed:
Create deploy.yml

Deployment is updated when changes are pushed to the main branch

malintheier committed:
Update README.md

sulenchy committed:
Merge pull request #777 from NoroffFEU/deploy-github-pages-735

Updates to "develop" regarding GitHub Pages

sulenchy committed:
Update README.md

sulenchy committed:
Merge pull request #803 from NoroffFEU/sulenchy-patch-1

Update README.md

acenyte committed:
Fix repo
